include/vcpkg/base/messages.h

```diff
@@ -2091,5 +2091,168 @@ namespace vcpkg
+    DECLARE_MESSAGE(PortBugDebugShareDir,
+                    (),
+                    "'{CURRENT_PACKAGES_DIR}' should not be translated.",
+                    "/debug/share should not exist. Please reorganize any important files, then use\n"
+                    "file(REMOVE_RECURSE \"${CURRENT_PACKAGES_DIR}/debug/share\")");
+    DECLARE_MESSAGE(PortBugMissingFile,
+                    (msg::path),
+                    "",
+                    "The /{path} file does not exist. This file must exist for CMake helper ports.");
+    DECLARE_MESSAGE(PortBugMergeLibCMakeDir,
+                    (msg::spec),
+                    "",
+                    "The /lib/cmake folder should be merged with /debug/lib/cmake and moved to "
+                    "/share/{spec}/cmake.");
+    DECLARE_MESSAGE(PortBugUseVcpkgCMakeConfigFixup,
+                    (),
+                    "",
+                    "Please use the helper function `vcpkg_cmake_config_fixup()` "
+                    "from the port vcpkg-cmake-config.`");
+    DECLARE_MESSAGE(
+        PortBugMisplacedCMakeFiles,
+        (msg::spec),
+        "",
+        "The following cmake files were found outside /share/{spec}. Please place cmake files in /share/{spec}.");
+    DECLARE_MESSAGE(PortBugDllInLibDir,
+                    (),
+                    "",
+                    "The following dlls were found in /lib or /debug/lib. Please move them to /bin or "
+                    "/debug/bin, respectively.");
+    DECLARE_MESSAGE(PortBugMissingLicense,
+                    (msg::spec),
+                    "'{CURRENT_PACKAGES_DIR}' should not be translated.",
+                    "The software license must be available at ${CURRENT_PACKAGES_DIR}/share/{spec}/copyright");
+    DECLARE_MESSAGE(PortBugFoundCopyrightFiles, (), "", "The following files are potential copyright files:");
+    DECLARE_MESSAGE(PortBugFoundExeInBinDir,
+                    (),
+                    "",
+                    "The following EXEs were found in /bin or /debug/bin. EXEs are not valid distribution targets.");
+    DECLARE_MESSAGE(PortBugDllsWithNoExports, (), "", "The following DLLs have no exports:");
+    DECLARE_MESSAGE(PortBugSetDllsWithoutExports,
+                    (msg::env_var),
+                    "",
+                    "DLLs without any exports are likely a bug in the build script. If this is intended, add the "
+                    "following line in the portfile:\n"
+                    "SET({env_var} enabled)");
+    DECLARE_MESSAGE(RunningCommandFailed, (msg::command_line), "", "Running command:{command_line}\nFailed:");
+    DECLARE_MESSAGE(PortBugDllAppContainerBitNotSet,
+                    (),
+                    "",
+                    "The following DLLs do not have the App Container bit set:");
+    DECLARE_MESSAGE(PortBugRequiredDllAppContainerBit,
+                    (),
+                    "",
+                    "The App Container bit must be set for Windows Store apps.");
```

Comment:
Ditto should be combined.

```suggestion
    DECLARE_MESSAGE(PortBugDllAppContainerBitNotSet,
                    (),
                    "",
                    "The App Container bit must be set for Windows Store apps. The following DLLs do not have the App Container bit set:");
```